ps0/practice_0.py

```diff
@@ -0,0 +1,11 @@
+import math
+
+
+def main():
+    x = int(input('Enter number x: '))
+    y = int(input('Enter number y: '))
+    print('x**y =', (x ** y))
+    print('log(x) =', math.log(x))
+
+
+main()
```

Comment:
```suggestion
if __name__ == "__main__":
    main()
```

[This link](https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do/) explains what the suggestion does and why it is preferable to directly invoking the `main()` method.